### ISSUE TITLE
Distribution of software

### ISSUE THREAD

**Issue by NoeelMoeskops** [stephendpmurphy/mpsse-cli#44]
Not for right now. But in the long run it might be worthwhile to think about how to distribute the software. I favour Flatpak since it is cross-platform on most (if not all) Linux platforms. And that way we just package it once and be done with it (along with some USB access rules that should be interesting). Although distribution via `dnf` or `apt` would also be really cool. But that would require more maintenance and audits to get it into the main repo's. What do you think @stephendpmurphy?

**Comment by stephendpmurphy** [comment]
I agree! I say we shoot for as many distribution methods as possible. We can go with flatpak first if you are already familiar with it. 👍
